Doc/whatsnew/3.8.rst

```diff
@@ -405,17 +405,17 @@ Other Language Changes
 
   (Contributed by Jonathan Eunice and Serhiy Storchaka in :issue:`30688`.)
 
-* Dict and dictviews are now iterable in reversed insertion order using
-  :func:`reversed`. (Contributed by Rémi Lapeyre in :issue:`33462`.)
+* :class:`dict` and dictionary views are now iterable in reversed insertion
+  order using :func:`reversed`. (Contributed by Rémi Lapeyre in :issue:`33462`.)
 
 * The syntax allowed for keyword names in function calls was further
   restricted. In particular, ``f((keyword)=arg)`` is no longer allowed. It was
   never intended to permit more than a bare name on the left-hand side of a
   keyword argument assignment term. See :issue:`34641`.
 
 * Generalized iterable unpacking in :keyword:`yield` and
-  :keyword:`return` statements no longer requires enclosing parentheses.
-  This brings the *yield* and *return* syntax into better agreement with
+  :keyword:`return` statements no longer requires enclosing parenthesis.
```

Comment:
This looks incorrect: things like `thesis` have plural form `theses`

Reply:
Oh you're correct. I was under the impression that it was `parenthesis` for a single pair of `()` and parentheses referred to multiple pairs of `()`. Apparently, parenthesis refers a single `(` and parentheses refers to a pair `()`. It seems like I was using the wrong form of the word all this time and hadn't noticed, good to know. (:

Reply:
Reverting this change.
```suggestion
  :keyword:`return` statements no longer requires enclosing parentheses.
```